app/(tabs)/send.tsx

```diff
@@ -1,17 +1,39 @@
-import { Feather } from '@expo/vector-icons';
-import { View } from 'react-native';
+import { View, Alert } from 'react-native';
+import React, { useRef } from 'react';
 
 import { HeaderBar } from '~/components/HeaderBar';
-import FText from '~/components/Text/FText';
 import { Frame } from '~/components/Wrappers/Frame';
 
+import RecipientInput from '~/components/Send/RecipientInput';
+import AmountInput from '~/components/Send/AmountInput';
+import { Button } from '~/components/Button';
+
 export default function Send() {
+  const recipientRef = useRef<{ value: string } | null>(null);
+  const amountRef = useRef<{ value: string } | null>(null);
```

Comment:
Is there a reason why you are using `useRef` here and `forwardRef` for the input components?